packages/feedback-spa/src/app/feedback/home/feedback-home.component.ts

```diff
@@ -51,7 +51,9 @@ export class FeedbackHomeComponent implements OnInit {
             }
           })[0];
           if (fbIssue) {
+            // tslint:disable-next-line: no-string-literal
```

Comment:
Do not disable the tslint rule, find out the reason behind the error/warning and fix it.